extensions/amp-story-interactive/0.1/amp-story-interactive-img-quiz.css

```diff
@@ -15,11 +15,13 @@
  */
 
 .i-amphtml-story-interactive-img-quiz-answer-choice {
+  --i-amphtml-interactive-answer-choice-border: 0.125em solid
+    var(--i-amphtml-interactive-option-background-color) !important;
   position: absolute !important;
   height: 1.5em !important;
   width: 1.5em !important;
   bottom: 0.5em !important;
-  border: 0.125em solid var(--i-amphtml-interactive-option-background-color) !important;
+  border: var(--i-amphtml-interactive-answer-choice-border) !important;
```

Comment:
Would this be fixed if we remove line 24? I'm not sure if it's still useful.

That way we wouldn't need to override [--i-amphtml-interactive-answer-choice-border](https://github.com/ampproject/amphtml/blob/main/extensions/amp-story/1.0/amp-story.css#L914)